Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -462,7 +585,7 @@ public Apis.Bigquery.v2.Data.Job DoCopy()
                             {
                                 DestinationTable = Destination,
                                 SourceTable = Source,
-                                WriteDisposition = WriteMode.ToString()
+                                WriteDisposition = (WriteMode == null) ? WriteDisposition.WriteIfEmpty.ToString() : WriteMode.ToString()
```

Comment:
Got it!  That's a pretty cool convention.  Didn't realize that could be used in between an object and a method call.